juntagrico/templates/size_change.html

```diff
@@ -5,34 +5,73 @@
 {% block page_title %}
     {% vocabulary "subscription" as v_subscription %}
     <h3>
-        {% blocktrans %}{{ v_subscription }}-Grösse ändern{% endblocktrans %}
+        {% blocktrans %}{{ v_subscription }}-Bestandteile ändern{% endblocktrans %}
     </h3>
 {% endblock %}
 {% block content %}
     {% config "enable_shares" as c_enable_shares %}
     {% vocabulary "subscription" as v_subscription %}
     {% vocabulary "subscription_pl" as v_subscription_pl %}
     {% vocabulary "share_pl" as v_share_pl %}
-    {% if saved %}
-        <div class="alert alert-success">
-            {% blocktrans %}Die {{ v_subscription }}-Grösse wurde erfolgreich geändert.{% endblocktrans %}
-            <br/>
-            <a href="{% url 'sub-detail' %}" class="alert-link">
-                {% blocktrans %}Zurück zur {{ v_subscription }}-Übersicht{% endblocktrans %}
-            </a>
-        </div>
-    {% else %}
-        {% crispy form %}
 
-        {% if subscription.state != 'canceled' and  subscription.state != 'inactive' %}
-            <div class="row mb-3">
-                <div class="col-md-12">
-                    <a href="{% url 'sub-cancel' subscription.id %}"class="btn btn-danger">
-                        {% blocktrans %}{{ v_subscription }} komplett künden{% endblocktrans %}
-                    </a>
-                </div>
+    <div class="row mb-3">
+        <h4>{% trans "Bestandteile verwalten" %}</h4>
+    </div>
+    <div class="row">
+        <table class="table subscription-table">
```

Comment:
Might add the start date of each subscription here? Especially if it starts in the future.
Maybe cross out subscriptions that have been deactivated and or show end date? Or don't show them at all?

Reply:
yes here the active parts should be used not all the parts